CODEOWNERS

```diff
@@ -1,37 +1,194 @@
+#   Ensure the backwards-compatibility of Cilium REST and gRPC APIs, including
+#   for Hubble flow export.
+# - @cilium/build:
+#   Provide feedback on languages and scripting used for build and packaging
+#   system: Make, Shell, Docker.
+# - @cilium/cli:
+#   Provide user experience feedback on changes to Command-Line Interfaces.
+#   These owners are a stand-in for the user community to bring a user
+#   perspective to the review process. Consider how information is presented,
+#   consistency of flags and options.
+# - @cilium/ci-structure:
+#   Guidance around the best use of Cilium project continuous integration and
+#   testing infrastructure, including GitHub actions, VM helpers, testing
+#   frameworks, etc.
+# - @cilium/contributing:
+#   Encourage practices that ensure an inclusive contributor community. Review
+#   tooling and scripts used by contributors.
+# - @cilium/docs-structure:
+#   Ensure the consistency and layout of documentation. General feedback on the
+#   use of Sphinx, how to communicate content clearly to the community. This
+#   code owner is not expected to validate the technical correctness of
+#   submissions. Correctness is typically handled by another code owner group
+#   which is also assigned to any given piece of documentation.
+# - @cilium/github-sec:
+#   Responsible for maintaining the security of repositories in the Cilium
+#   project by maintaining best practices for workflow usage, for instance
+#   preventing malicious use of GitHub actions.
+# - @cilium/helm:
+#   Provide input on the way that Helm can be used to configure features. These
+#   owners are a stand-in for the user community to bring a user perspective to
+#   the review process. Ensure that Helm changes are defined in manners that
+#   will be forward-compatible for upgrade and follow best practices for
+#   deployment (for example, being GitOps-friendly).
+# - @cilium/metrics:
+#   Provide recommendations about the types, names and labels for metrics to
+#   follow best practices. This includes considering the cardinality impact of
+#   metrics being added or extended.
+# - @cilium/tophat:
+#   Top Hat duties rotate between the committer group from week to week, and
+#   they may assist in maintenance, triage and backporting duties across
+#   different Cilium repositories. Catch-all for code not otherwise owned by a
+#   team.
+# - @cilium/vendor:
+#   Review vendor updates for software dependencies to check for any potential
+#   upstream breakages / incompatibilities. Discourage the use of unofficial
+#   forks of upstream libraries if they are actively maintained.
+#
+# Repository Owners
+# +++++++++++++++++
+#
+# The following code owners are responsible for a range of general feedback for
+# specific repositories:
+#
+# - @cilium/hubble-ui:
+#   Maintains the Hubble UI graphical interface.
+# - @cilium/sig-agent:
+#   Provides Cilium (agent) general Go review. Internal architecture, core data
+#   structures and daemon startup.
```

Comment:
This team IMO isn't really a repository owner like the other two. Maybe it's better mentioned in the _Cilium Internals_ section below, where also all the other `@cilium/sig-*` teams' roles are described.